lib/guard/rack/runner.rb

```diff
@@ -76,8 +61,8 @@ def kill(pid, force = false)
     end
 
     def run_rack_command!
-      command = build_rack_command
-      UI.debug("Running Rack with command: #{command.inspect}")
+      command = Guard::Rack::Command.new(options)
+      UI.debug("Running Rack with command: #{command}")
       spawn(*command)
```

Comment:
I is not `Process#spawn` it is custom method above that call `Spoon.spawnp(*args)`
```ruby
# from Spoon gem
def self.spawnp(*args)
  posix_spawnp(args[0], nil, nil, args, ENV)
end
```
So for `String` command instead of 
`posix_spawnp('rackup, nil, nil, [args], env')`
we run
`posix_spawnp('r', nil, nil, 'rackup ..', ENV)`